utils/style_doc.py

```diff
@@ -367,21 +367,17 @@ def style_docstring(docstring, max_len):
     return "\n".join(new_lines), "\n\n".join(black_errors)
 
 
-def style_file_docstrings(code_file, max_len=119, check_only=False):
+def style_docstrings_in_code(code, max_len=119):
```

Comment:
Splitting this function in two so I have one I can use to format code in `check_copies`.